### ISSUE TITLE
How to use a fine-tuned model after training (DUA)

### ISSUE THREAD

**Issue by Jamil-Palma** [SWivid/F5-TTS#931]
### Checks

- [x] This template is only for usage issues encountered.
- [x] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [x] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [x] I am using English to submit this issue to facilitate community communication.

### Environment Details

Hi everyone,

I just finished fine-tuning my model using DUA. The training completed successfully:

Epoch 100/100: 100%|█████████████████████████████████████████████████████████████████| 24/24 [00:14<00:00, 1.63update/s, loss=1.32, update=2400]
Saved last checkpoint at update 2400

However, I’m not sure how to use the fine-tuned model now.
I found the saved checkpoint using:

find ./ckpts -name "checkpoint_last.pth"

But it’s not clear how to load this model and use it for inference. Could someone guide me on the next steps?

I'm also sharing my training config (config.yaml) in case it helps.

Thanks in advance!

### Steps to Reproduce

training with this configuration
root@12501623ba9f:/workspace/F5-TTS# cat src/f5_tts/configs/F5TTS_LJS_Min.yaml
hydra:
  run:
    dir: ckpts/${model.name}_${model.mel_spec.mel_spec_type}_${model.tokenizer}_${datasets.name}/${now:%Y-%m-%d}/${now:%H-%M-%S}

datasets:
  name: LJSpeech
  batch_size_per_gpu: 2400
  batch_size_type: frame
  max_samples: 64
  num_workers: 4
  dataset_dir: /workspace/F5-TTS/data/LJSpeech_char
  vocab_path: /workspace/F5-TTS/data/LJSpeech_char/vocab.txt

model:
  name: F5TTS_v1_Base
  tokenizer: char
  tokenizer_path: /workspace/F5-TTS/data/LJSpeech_char/vocab.txt
  backbone: DiT
  arch:
    dim: 1024
    depth: 22
    heads: 16
    ff_mult: 2
    text_dim: 512
    text_mask_padding: True
    qk_norm: null
    conv_layers: 4
    pe_attn_head: null
    checkpoint_activations: False
  mel_spec:
    target_sample_rate: 24000
    n_mel_channels: 100
    hop_length: 256
    win_length: 1024
    n_fft: 1024
    mel_spec_type: vocos
  vocoder:
    is_local: False
    local_path: null

ckpts:
  logger: null
  log_samples: True
  save_per_updates: 1000
  keep_last_n_checkpoints: -1
  last_per_updates: 500
  save_dir: ckpts/${model.name}_${model.mel_spec.mel_spec_type}_${model.tokenizer}_${datasets.name}
optim:
  epochs: 100
  learning_rate: 1e-5
  num_warmup_updates: 20000
  grad_accumulation_steps: 1
  max_grad_norm: 1.0
  bnb_optimizer: False
root@12501623ba9f:/workspace/F5-TTS# 

I use this command to start
accelerate launch --mixed_precision=fp16 src/f5_tts/train/train.py \
--config-name F5TTS_LJS_Min.yaml \
++training.save_model=true \
++model.use_ema=false

### ✔️ Expected Behavior

generate audio with new model trained

### ❌ Actual Behavior

I don't know how use this trained model

**Comment by SWivid** [comment]
`f5-tts_infer-cli --help`

**Comment by SWivid** [comment]
some points confused me:
- > find ./ckpts -name "checkpoint_last.pth"
   
    we don't save with .pth
- > ++training.save_model=true
++model.use_ema=false

    what do these for?
